src/utils/github_updater.rs

```diff
@@ -171,12 +209,9 @@ impl GithubUpdater {
             last_commit: response.pushed_at,
         })
     }
-}
 
-fn get_github_path(url: &str) -> Option<String> {
-    let re = Regex::new(r"https?://github\.com/([\w\._-]+)/([\w\._-]+)").unwrap();
```

Comment:
Why move the regex out of here? If you want to avoid recompilation of it it should be `static Lazy<Regex>`, not a field.

Reply:
I changed it to where the github updater is persistent, so a `Lazy` wasn't even needed. It's a small thing, but it was easy enough to not recompile the regex so I did